src/main/java/com/team2/hrbank/changelog/dto/EmployeeLogDto.java

```diff
@@ -11,49 +11,56 @@ private EmployeeLogDto() {
     }
 
     @Builder
-    public record EmployeeBaseInfo(
+    public record CreateLogRequest(
+            // Employee info
             LocalDateTime hireDate,
             String name,
             String position,
             String department,
             String email,
             String employeeNumber,
-            String status
-    ) {}
-
-    public record LogMetadata(
-            @Nullable
-            String memo,
+            String status,
+            // metadata
+            @Nullable String memo,
             String ipAddress
-    ) {
-
-        public static LogMetadata withMemo(String ipAddress, String memo) {
-            return new LogMetadata(memo, ipAddress);
-        }
-
-        public static LogMetadata withoutMemo(String ipAddress) {
-            return new LogMetadata(null, ipAddress);
-        }
-
-    }
-
-    @Builder
-    public record CreateLogRequest(
-            EmployeeBaseInfo employeeInfo,
-            LogMetadata metadata
     ) {}
 
     @Builder
     public record CreateUpdateLogRequest(
-            EmployeeBaseInfo before,
-            EmployeeBaseInfo after,
-            LogMetadata metadata
+            // Before Employee info
+            LocalDateTime beforeHireDate,
+            String beforeName,
+            String beforePosition,
+            String beforeDepartment,
+            String beforeEmail,
+            String beforeEmployeeNumber,
+            String beforeStatus,
+            // After Employee info
+            LocalDateTime afterHireDate,
+            String afterName,
+            String afterPosition,
+            String afterDepartment,
+            String afterEmail,
+            String afterEmployeeNumber,
+            String afterStatus,
+            // metadata
+            @Nullable String memo,
+            String ipAddress
     ) {}
```

Comment:
Before, After를 생성하는 로직은 ChangeLogService의 책임으로 생각됩니다. 따라서 다음과 같은 코드가 적절해보입니다.

```suggestion
    public record CreateUpdateLogRequest(
        String employeeNumber, // 대상 직원 사번
        @Nullable String memo, // 메모
        
        // 변경 상세 내용
        @Nullable String employeeName, // 직원 이름
        @Nullable String employeeEmail, // 직원 이메일
        @Nullable String departmentName, // 부서 이름
        @Nullable String employeePosition, // 직원 직함
        @Nullable LocalDate employeeHireDate, // 직원 입사일
        @Nullable String employeeStatus, // 직원 상태
        
        String ipAddress, // IP 주소
        LocalDateTime at // 시간
    ) {}
```